.github/workflows/integration.yaml

```
@@ -0,0 +1,34 @@
+name: ops Integration Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '42 7 25 * *'
+
+permissions: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        preset: ['k8s', 'microk8s']
```

> The current approach (without groups) fails repeatedly ... when running the second test out of the two.

Worth adding a matrix to `-k` the two individual tests? I realise that adds extra work when adding a new integration test, but maybe it's fine.

```suggestion
        preset: ['k8s', 'microk8s']
        test: ['test_direct_connection', 'test_with_tls']
```

---

I think Dima's right that it's not worth this for just two tests. We can always rearrange as we add more.

---

The motivation for my suggestion is purely as a fix for Dima's comment:

> The current approach (without groups) fails repeatedly ... when running the second test out of the two.
> I'm not quite sure how to debug that, and I wonder if perhaps running two tests in series just somehow uses up all the available disk/volume space or something?

I agree that this isn't worthwhile for just two tests otherwise.

---

I've kicked off a test with `-k` in a separate branch. Fingers crossed.

---

works, finally! 😓